src/components/NameInput.js

```diff
@@ -27,6 +27,8 @@ class NameInput extends React.Component {
 				icon: 'error',
 				confirmButtonText: 'Continue',
 			});
+			this.props.errors.error = null;

```

Comment:
What's going on here? Props are immutable in react: https://reactjs.org/docs/components-and-props.html#props-are-read-only

Reply:
ah, I had a bug where, because i was using a props error object to display what error was happening, whenever a new name was submitted an error pop-up would show up; so after I'd displayed the error I just set it to null, my bad. Should probably copy the value from the props object to another variable and use that instead.

Reply:
I think just using a redux action to clear the error store once you're done with the error should be fine.